### DISCUSSION TITLE
ＡＩロボットによるコメントは禁止・無条件削除すべきです

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#457] [要望・アイデア・不具合]

**Comment by boc1211** [comment]
確かに先日から急に現れたので気になってました。
アカウント名的に公式が実験してるのかと思ってたんですが、それにしてはアナウンスも無いしな、と・・・。

第三者がやってることならスパムと見分けが付かないので一律禁止に同意したいです。
公式がやってることなら何点か改善点はある(botと明記する・いいねはしない)ものの、個人的には有ってもいいかなと思ってます。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
仰っている内容は、理屈としても感情としても概ね理解できます。
しかしながら、AIを使用した自動投稿かどうかは、そのように見えるだけで実際には違う場合もあり
無条件削除等は、関係ないユーザーさんの投稿を削除してしまう可能性もあるので、運営的に難しいです、、、

個別に対応しておりますので、特定のアカウントなど怪しいアカウントを見つけられた際は
Discussionsではなく、通報にてご連絡をお願いします🙇
[【初めにお読みください】Qiita Discussions へようこそ](https://github.com/increments/qiita-discussions/discussions/116)
